### DISCUSSION TITLE
Type Coverage - Output only files with problems

### DISCUSSION THREAD

**Discussion by aon4o** [pestphp/pest#1183] [Ideas]
I have a Laravel 11 project where I use the type-coverage plugin.
The project has many files, so when I run the type coverage test I have to scroll through too many files just to fix 5 or 6 missed types.
I think it would be cool if a flag was available that tells the plugin to output only the files with problems.

**Comment by troccoli** [comment]
I don't use type coverage, just code coverage, but I would still like Pest to show files that are not 100% covered or below the minimum, if specified.

I have recently started a new project, with only 45 files, and it is already very inconvenient to scroll up to see which files are below my minimum coverage. It will be quite bad when the project grows.

**Comment by aon4o** [comment]
Somebody has already implemented it.

https://github.com/pestphp/pest-plugin-type-coverage/pull/33

**Reply by troccoli** [reply]
Thanks, but as I said I don't use type coverage, so I don't have the `--terse` option.
